just/code_check.just

```diff
@@ -0,0 +1,23 @@
+# Formats the code
+[group('code_check')]
+fmt args="":
+  cargo fmt --all {{args}}
```

Comment:
Actually our rustfmt.toml still requires nightly, so I wouldn't keep this

Reply:
This is the common fmt logic used by all of our projects. It ignores the nightly only features